samples/global.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Lets get rid of the global.json; we shouldn't need it.

Reply:
I can delete it though since I was at least able to isolate the issue. Let me know and I'll take the appropriate action.

Reply:
I think it'd be best to remove it